### ISSUE TITLE
ZencoderClient.listJobs can't parse response from server

### ISSUE THREAD

**Issue by ftrofin** [zencoder/zencoder-java#9]
Today I came across this exception while trying to list the jobs:

```java
com.brightcove.zencoder.client.ZencoderClientException: Unable to deserialize ZencoderCreateJobResponse as JSON
Caused by: org.codehaus.jackson.JsonParseException: Numeric value (3169615172842) out of range of int
 at [Source: java.io.StringReader@640c52ba; line: 1, column: 73227]
	at org.codehaus.jackson.JsonParser._constructError(JsonParser.java:1433)
	at org.codehaus.jackson.impl.JsonParserMinimalBase._reportError(JsonParserMinimalBase.java:521)
	at org.codehaus.jackson.impl.JsonParserBase.convertNumberToInt(JsonParserBase.java:829)
	at org.codehaus.jackson.impl.JsonParserBase.getIntValue(JsonParserBase.java:625)
	at org.codehaus.jackson.map.deser.std.StdDeserializer._parseInteger(StdDeserializer.java:303)
	at org.codehaus.jackson.map.deser.std.StdDeserializer$IntegerDeserializer.deserialize(StdDeserializer.java:819)
	at org.codehaus.jackson.map.deser.std.StdDeserializer$IntegerDeserializer.deserialize(StdDeserializer.java:806)
	at org.codehaus.jackson.map.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:299)
	at org.codehaus.jackson.map.deser.SettableBeanProperty$MethodProperty.deserializeAndSet(SettableBeanProperty.java:414)
	at org.codehaus.jackson.map.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:697)
	at org.codehaus.jackson.map.deser.BeanDeserializer.deserialize(BeanDeserializer.java:580)
	at org.codehaus.jackson.map.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:299)
	at org.codehaus.jackson.map.deser.SettableBeanProperty$MethodProperty.deserializeAndSet(SettableBeanProperty.java:414)
	at org.codehaus.jackson.map.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:697)
	at org.codehaus.jackson.map.deser.BeanDeserializer.deserialize(BeanDeserializer.java:580)
	at org.codehaus.jackson.map.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:299)
	at org.codehaus.jackson.map.deser.SettableBeanProperty$MethodProperty.deserializeAndSet(SettableBeanProperty.java:414)
	at org.codehaus.jackson.map.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:697)
	at org.codehaus.jackson.map.deser.BeanDeserializer.deserialize(BeanDeserializer.java:580)
	at org.codehaus.jackson.map.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:217)
	at org.codehaus.jackson.map.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:194)
	at org.codehaus.jackson.map.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:30)
	at org.codehaus.jackson.map.ObjectMapper._readMapAndClose(ObjectMapper.java:2732)
	at org.codehaus.jackson.map.ObjectMapper.readValue(ObjectMapper.java:1877)
	at com.brightcove.zencoder.client.ZencoderClient.listJobs(ZencoderClient.java:218)
```

Either the server is returning garbage or the code running on the server was changed and the SDK is out of sync.

**Comment by mmcc** [comment]
Thanks @ftrofin, we're looking into it. It might be a bit delayed getting an update pushed to Sonatype, but we can push a fix and give you a jar in the meantime.

**Comment by ftrofin** [comment]
Great, thank you Matthew! Please let me know when you have the fix available, I am in dire need of it!
Just curious: Is the problem in the SDK or does the server return garbage and the JSON parser chokes on it?

**Comment by mmcc** [comment]
We think it's a problem with the SDK and it should be fixed by just using a long int instead, but we want to look more into the request itself (that number seems way too big to be an ID from us). Mind shooting me an email so we can get some more information about the request itself? mmcclure at brightcove

**Comment by skidder** [comment]
@ftrofin I've been unable to reproduce this error when listing jobs associated with your account.  Would you please provide the following details in an email?
 * JDK version
 * Jackson version
 * Time and date when you invoked the 'listJobs' API

Please send them to me, Scott Kidder, at the following:
skidder at brightcove dot com

**Comment by skidder** [comment]
Thanks @ftrofin , we were able to reproduce this issue and determine that the parsing error was due to the video-bitrate field in the listJobs API response containing a value that was too large to fit in a 32-bit integer field declared in Java binding class.  It's also worth noting that the extremely large bitrate value was associated with an job input that Zencoder marked as invalid.

I'll fix this issue by change the video and audio bitrate fields in the ZencoderMediaFile class to be type Long instead of type Integer.  This will ensure the response data in the client matches what's reported by Zencoder.
